### PR TITLE
Linters Setup

### PR DESCRIPTION
**Pull request by Othniel2471**
### In this branch i : 

- Added linters.
- Components, pages, and styles folders.

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Home = () => (
+  <div>
+    <h1>Home</h1>
+    <p>Book Store</p>
+  </div>
+);
```
Comment (brytebee) [inline src/pages/Home.js]: This component which serves as both the Home and Books component is required to render a static list of books.

Kindly create and map over an array to render them to the page.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Othniel2471 👋,

##

### Highlights💡
- Project correctly set up.✔️
- Good commit message.✔️ 
- Good PR Summary.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.js (hunk 1 above)
Comment (Othniel2471) [reply src/pages/Home.js]: @brytebee  please is it going to be dynamic, or can I just create an object of books and then render it, or am I meant to write the whole functionality for adding the books dynamically?

Comment (brytebee) [reply src/pages/Home.js]: @Othniel2471 It can be a static book such as BookList. Then just map over it to render them on the page.

**Review by uwadonat — APPROVED** [review]
## Hi @Othniel2471,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Well-organized and clean project ✔️
- Correct workflow was used ✔️
- Professional readme file ✔️


209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196

# Status: Approved ✔️✔️✔️

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Othniel2471** [comment]
@uwadonat Thank you for the review

**Pull request merged**